draft-ietf-rats-architecture.md

```diff
@@ -610,6 +610,14 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the attestation evidence is conveyed. As an example, if the device and Verifier
+communicate over a link that is established using a Root of Trust (e.g. cellular
+communications link where authentication is tied to the presence of a secure element),
+then the Verifier may be able to trust the attestation evidence from the device without
+an additional endorsement or even a cryptographically-verifiable signature of the 
+evidence.
```

Comment:
So you mean the Evidence isn't signed or encrypted?

Reply:
Endorsements are signed (integrity protected and authenticated) by supply chain entity such as a vendor or manufacturer. Evidence is signed by an attestation key that is on the attesting device. The roles architecture distinguishes between Endorsers and Attesters for good reasons even though there can be cases where it isn't that clear that they are different. I don't think this section is trying to draw attention to one of those corner cases. Therefore, it appears the term Endorsement was used by mistake when Evidence was intended.

Reply:
@nedmsmith 

That wasn't really the intention behind the cited text.  If the communications link is already trusted, then an endorsement may have been used in establishing the communications link.  In that case, an additional endorsement for attestation evidence is not required.

Reply:
I agree with the three sentences above.

I think the text at line 616 may be what is the problem because it presumes knowledge of the secure element context is existential. The Verifier needs to be told that the root-of-trust / secure element environment exists. It can happen manually, such as an IT department physically issuing a smart card or smart device with a SE to a user, generating the key pair, update the Verifier to trust the public that was generated. Manual approaches work in IT environments where employees are subject to IT procedures. Manual processes may not scale to Internet ecosystems. Remote attestation is trying to go beyond manual processes by generating keys in devices by manufacturers and issuing machine certs that establish that a key is protected by a secure element. This seems like a basic idea that if it isn't clear might belong in the introduction?